string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,12 @@
+/* exported capitalizeWord */
+function capitalizeWord(word) {
+  const newWord = word.toLowerCase();
+  const capWord = newWord.charAt(0).toUpperCase() + newWord.slice(1);
+  for (var i = 0; i < capWord.length; i++) {
+    if (capWord[i] === 's') {
+      const finalCase = capWord.charAt(i).toUpperCase() + capWord.slice(i + 1);
+      return capWord.substring(0, 4) + finalCase;
+    }
+  }
```

Comment:
This seems very fragile. Why would all words containing an `'s'` need to be sliced up like this? Since `'Javascript'` is a special case, just check if the work `=== 'Javascript'` and then return `'JavaScript`.